src/syscheckd/src/db/src/dbRegistryKey.cpp

```diff
@@ -88,6 +101,8 @@ void RegistryKey::createJSON()
     data["user_name"] = m_username;
     data["group_name"] = m_groupname;
     data["mtime"] = m_time;
+    data["hash_full_path"] = m_hashpath;
+
```

Comment:
This space is unnecessary!

Reply:
Its just for clearly reading of the code, the next line insert the json object "data" into the json object "conf"